userguide/snippets/pathlengths.rst

```diff
@@ -34,11 +34,27 @@ can also reduce the length of these limits by one or more bytes.
    | Names               |                |                                                                        |
    +---------------------+----------------+------------------------------------------------------------------------+
    | Mounted Filesystem  | 88 bytes       | Mounted filesystem path length (*MNAMELEN*). Longer paths can prevent  |
-   | Paths               |                | a device from being mounted.                                           |
+   | Paths               |                | a device from being mounted.  See below for examples of path length    |
+   |                     |                | calculation.                                                           |
    +---------------------+----------------+------------------------------------------------------------------------+
    | Device Filesystem   | 63 bytes       | `devfs(8)                                                              |
    | Paths               |                | <https://www.freebsd.org/cgi/man.cgi?query=devfs&sektion=8>`__ device  |
    |                     |                | path lengths (*SPECNAMELEN*). Longer paths can prevent a device from   |
    |                     |                | being created.                                                         |
    +---------------------+----------------+------------------------------------------------------------------------+
 
+Care is needed with regard to ZFS snapshots. **If the mounted path length for a snapshot exceeds 88 characters, 
+then the snapshot and its data will be safe but inaccessible until its mounted path length is reduced**. 
+Typically this is rectified by renaming (shortening) the dataset, snapshot, or mountpoint, so that the path length is
+below the 88 byte limit. Note that ZFS snapshots are automatically mounted on demand within hidden :file:`/.zfs/snapshot/`
+directories, and can also be manually mounted using the command line. The 88 byte limit affects both types of mount.
```

Comment:
s/using/from/

Reply:
done

Reply:
There is only 1 .zfs/snapshot directory. This needs to be reworded to refer to the multiple snapshots in the snapshot directory. The path given is /.zfs/snapshot Is this correct, the zfs/snapshot dir is off of root? 
88 byte/char issue here again. Also "both types of mount" doesn't make sense to me and seems redundant if I understand what is trying to be said here. The way something is mounted doesn't affect the path to the mount.

Reply:
There is only 1 .zfs/snapshot dir **per dataset**, as a result there can be many in the pool. In each case it hangs directly off the root of that dataset. The updated text after Warren's review reads: *"The mountpoint used will be a hidden :file:`.zfs/snapshot/{name}` directory within the same ZFS dataset"*

Please reread the updated version after Warren's review, which seems to address most of the points you made.